### ISSUE TITLE
Forbid PROJECT_PATH empty if PROJECT_TYPE=local

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#816]
In GitLab by @mcastril on May 17, 2022, 18:45

To avoid that the root dir '/' is copied inside the 'proj' dir.

@pabretonniere

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on May 19, 2022, 08:59

I think this is already done in 3.14, but will check

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jul 19, 2022, 14:28

There were two bugs. 

- Project destination was always ignored on svn and git. It was taking the name of the git/svn repo
- Project destination was allowed to be empty. 

Both are fixed on 3.14 and 4.0.0, now autosubmit will raise an critical error if this parameter is empty except if the project_type is none ( in that case there is no proj as all templates are internal)

**Comment by kinow** [comment]
In GitLab by @mcastril on Aug 8, 2022, 16:09

I realized there was a mistake in this issue's title (and therefore everything).

The point was that there were experiments declared as local projects, non defining the project path. Then copying '/'  to the experiment's directory.

For this purpose, I think `PROJECT_DESTINATION` is irrelevant.

Now I have this issue:

` [CRITICAL] Autosubmit couldn't identify the project destination. [eCode=7014]`

With

```
[project]
# Select project type. STRING = git, svn, local, none
# If PROJECT_TYPE is set to none, Autosubmit self-contained dummy templates will be used
PROJECT_TYPE = none
# Destination folder name for project. type = STRING, default = leave empty,
PROJECT_DESTINATION =
```

But apparently this is allowed:

> Both are fixed on 3.14 and 4.0.0, now autosubmit will raise an critical error if this parameter is empty except if the project_type is none ( in that case there is no proj as all templates are internal)

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 8, 2022, 16:39

I think I was thinking in project_path already. This is the part that was already fixed

The project destination you're right about is bugged in 3.14.0b, I'm applying a fix and also, if the project_destination is not set, the folder will be named as the git/svn_repo last name or last local folder name, if it doesn't work either, it will be named as "project_files"
